Services/Notifications/classes/Setup/ilNotificationUpdateSteps.php

```diff
@@ -0,0 +1,116 @@
+<?php declare(strict_types = 1);
+/* Copyright (c) 1998-2021 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+class ilNotificationUpdateSteps implements ilDatabaseUpdateSteps
+{
+    protected \ilDBInterface $db;
+
+    public function prepare(ilDBInterface $db)
+    {
+        $this->db = $db;
+    }
+    
+    public function step_1() : void
+    {
+        include_once('./Services/Migration/DBUpdate_3560/classes/class.ilDBUpdateNewObjectType.php');
+        ilDBUpdateNewObjectType::addAdminNode('nota', 'Notification Service Administration Object');
```

Comment:
If this PR is merged after https://github.com/ILIAS-eLearning/ILIAS/pull/3925 , you'll have to use the `ilTreeAdminNodeAddedObjective` instead.

Reply:
Ok i will leave this thread open until the final merge